beacon-chain/core/blocks/benchmarks_test.go

```diff
@@ -0,0 +1,492 @@
+package blocks_test
+
+import (
+	"context"
+	"io/ioutil"
+	"math"
+	"strconv"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/prysmaticlabs/go-ssz"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/blocks"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/state"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/bitutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	"github.com/prysmaticlabs/prysm/shared/trieutil"
+	"github.com/sirupsen/logrus"
+)
+
+var runAmount = 64
+var validatorNum = 65536
+var conditions = "SML"
```

Comment:
what is this?

Reply:
nvm i see it. Can you make this an argument to `setBenchmarkConfig()`? 

Set all of these to constant or you may have leaks in your test.